packages/compass-import-export/src/modules/import.ts

```diff
@@ -540,18 +782,14 @@ export const selectImportFileName = (fileName: string) => {
     dispatch: ThunkDispatch<RootImportState, void, AnyAction>,
     getState: () => RootImportState
   ) => {
-    let fileStats = {};
     try {
       const exists = await checkFileExists(fileName);
       if (!exists) {
         throw new Error(`File ${fileName} not found`);
       }
-      const stats = await getFileStats(fileName);
+      const fileStats = await getFileStats(fileName);
 
-      fileStats = {
-        ...stats,
-        type: mime.lookup(fileName),
```

Comment:
I noticed we don't actually use this as we go with whatever detectImportFile returns. The types don't even include it anyway.